foreman/foreman.spec

```diff
@@ -31,6 +31,14 @@ Source7: %{name}-plugins.repo
 Source8: %{name}.gpg
 BuildArch:  noarch
 
+Requires: %{name}-core = %{version}-%{release}
+Requires: %{name}-debug
+Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.5
```

Comment:
> Where do we depend on foreman in mail repo? Isn't the solution to put the tasks into the main repo?

There is no dependency on `foreman` in the main repo that I can recall, but repoclosure would fail.

Moving the dependency may be valid, but that's not part of the PR?